rrfs-test/validated_yamls/templates/obtype_config/aircar_specificHumidity_133.yaml

```diff
@@ -1,9 +1,12 @@
      - obs space:
```

Comment:
Line 108:
`             is_in: 130,131,135`

This is the `133` file. Does this mean we don't need the filter at that location?

Reply:
@delippi Could you comment on this? Is the rejection filter for other obs types needed in the 133 yaml?

Reply:
This is one of a few things I will update when I go back through the aircft/aircar yamls for my "Phase 2" validation. No need to worry about this right now. This isn't needed but also doesn't change anything. I will fix this in the next few weeks. We don't assimilate specifcHumidity from 130, 131, 135 which are aircft not aircar.

Reply:
@delippi This file only contains 133 obs, why we want to check whether it is 130, 131, 135?  I think we should remove this filter as it does not apply to 133.

Reply:
I understand it does nothing. But it will add one do-nothing filter in the JEDI runtime and it will confuse users.

Reply:
@guoqing-noaa, I don't think @SamuelDegelia-NOAA's PR is the place to make this change--he is just trying to update the ctest case which should just use the yamls "as is". The change should be made when I update the yamls which won't be that long. It isn't confusing, it is just unnecessary. I should've removed it before committing this file to start. These are comments that should be made when I submit a PR for the yamls.

Reply:
> @guoqing-noaa, I don't think @SamuelDegelia-NOAA's PR is the place to make this change--he is just trying to update the ctest case which should just use the yamls "as is". The change should be made when I update the yamls which won't be that long. It isn't confusing, it is just unnecessary. I should've removed it before committing this file to start. These are comments that should be made when I submit a PR for the yamls.

Do you mean only you can make changes to the yaml files under `validated_yaml`?

Reply:
I'm not saying I'm the ONLY one who can make changes to the yamls in `validated_yamls`. I just mean this seems out of scope of this PR and Sam shouldn't have to update those yamls. I was planning to make this and other updates when I do my Phase 2 testing on the aircft/aircar yamls which I will start tomorrow and probably finish by end of next week. I think it can wait. Does it cause any problems with the actual ctests?